server/main.py

```diff
@@ -49,14 +49,13 @@
 
 app = flask.Flask(__name__)
 app.config.update(
-    SECRET_KEY=constants.FLASK_SECRET_KEY,
     MAX_CONTENT_LENGTH=8 * 1024 * 1024,
     ALLOWED_EXTENSIONS=set(['png', 'jpg', 'jpeg', 'gif'])
 )
 
 app.config.update(
     {
-        "SECRET_KEY": constants.FLASK_SECRET_KEY,
+        "SECRET_KEY": cloud_secrets.get("flask_secret_key"),
```

Comment:
Is there a reason why you don't want the SECRET_KEY in the above app.config.update call?
I thought it'd be nice to keep this one for just the OIDC stuff.

Reply:
6 of 1 half a dozen of another?  It was in both places, that seemed unnecessary, and given that I was changing it I didn't think too much about which one I was removing it from.  I probably biased toward what you see here because config is a dictionary and I prefer the syntax for dictionaries as "key : value" over what's above that.

Reply:
ok

Reply:
@lizlooney I moved all of the properties into a single call to config.update() and added documentation so that a reader can tell what properties belong to what software component.  Also standardizes on syntax, and removes the repeated call.  Is this acceptable?